share/doc/homebrew/Formula-Cookbook.md

```diff
@@ -107,39 +56,38 @@ class Foo < Formula
 
   def install
     system "./configure", "--prefix=#{prefix}", "--disable-debug", "--disable-dependency-tracking"
-#   system "cmake", ".", *std_cmake_args
+    # system "cmake", ".", *std_cmake_args
```

Comment:
Are we agreed on this now? We've been going back and forth on whether or not to do `"cmake", *std_cmake_args` or `"cmake", ".", *std_cmake_args` for in-tree builds for a while.